packages/core/src/dispatcher.ts

```diff
@@ -39,6 +39,11 @@ const MAX_INTERVAL_WITHOUT_KEEPALIVE = 24 * 60 * 60 * 1000 // one day
 const IPFS_CACHE_SIZE = 1024 // maximum cache size of 256MB
 const IPFS_OFFLINE_GET_TIMEOUT = 200 // low timeout to work around lack of 'offline' flag support in js-ipfs
 const PUBSUB_CACHE_SIZE = 500
+const DEFAULT_RESPONSE_DELAY_MS = 1000 // 1 second
```

Comment:
Best to ask sergey but the syncTimeoutSeconds (the amount of time we wait for responses to our query is defaulted to3 seconds). So these numbers need to reflect this

Reply:
https://github.com/ceramicnetwork/js-ceramic/blob/90e77ee5b5bb14d3711c14ef6852bd5c9442f3ca/packages/core/src/state-management/repository.ts#L45 found here